HTML/mockup-corporate/js/task3.js

```diff
@@ -0,0 +1,48 @@
+import {elements} from './elements.js';
+import {patterns} from './patterns.js';
+
+const inputNameArr = [
+  elements.nameInput,
+  elements.emailInput,
+  elements.passwordInput,
+];
+
+const patternArr = [patterns.name, patterns.email, patterns.password];
+
+const validator = (input, pattern) => {
+  let inputValue = input.value;
+  let validation = pattern.test(inputValue);
+
+  input.classList.remove('input--invalid');
+
+  if (!validation) {
+    input.classList.add('input--invalid');
+    return false;
+  } else {
+    return true;
+  }
+};
+
+const validArr = [false, false, false];
+
+inputNameArr.forEach((el, ind) => {
+  el.addEventListener('input', () => {
```

Comment:
maybe it is possible to have just one listener using [event delegation](https://javascript.info/event-delegation), which is generally very useful technic, which based on [bubbling and capturing of events](https://javascript.info/bubbling-and-capturing)